tails_files/install.sh

```diff
@@ -21,6 +21,13 @@ DESKTOP=$HOMEDIR/Desktop
 ANSIBLE=$PERSISTENT/securedrop/install_files/ansible-base
 SSH_ALIASES=false
 
+# check for Tails 2.x
+source /etc/os-release
```

Comment:
Well, @conorsch, os-release is a simple file that contains two environment vars: TAILS_PRODUCT_NAME and TAILS_VERSION_ID. It's almost begging to be sourced instead of grepped. We can grep through if ya'll consider that safer. It's a file owned by root in the base system however, so I wasn't considering it as a source of malicious code.
The structure is like:
```
TAILS_PRODUCT_NAME="Tails"
TAILS_VERSION_ID="2.3"
````